src/styles/style.css

```diff
@@ -510,17 +520,15 @@ h6 {
 .home {
   display: flex;
   flex-direction: column;
-  justify-content: center;
   align-items: center;
-  height: 60vh;
+  height: calc(300px - var(--footer-height));
+  padding-top: 30px;
 }
 
 .tagline {
-  margin: 1rem;
+  /* margin: 1rem; */
```

Comment:
We could get rid of this to keep it tidy

Reply:
Done!